src/Evolution/Systems/ScalarWave/Tags.hpp

```diff
@@ -30,4 +30,24 @@ struct Phi : db::SimpleTag {
   using type = tnsr::i<DataVector, Dim, Frame::Inertial>;
   static std::string name() noexcept { return "Phi"; }
 };
+
+namespace Tags {
+/*!
+ * \brief Tag corresponding to one of the constraints of the
+ * scalar-wave system
+ * \details Tags corresponding to one- and two-index constraints of the
+ * scalar-wave system, and their diagnostically useful combinations.
+ * For details on how these are defined and computed, see
+ * `OneIndexConstraintCompute`, `TwoIndexConstraintCompute`
```

Comment:
I feel like the effort to use the same comment for multiple tags has resulted in a comment that is more complicated to understand than necessary.

Why not write a simpler comment like this...
```
\brief Tag for the one-index constraint of the ScalarWave system

For details on how this is defined and computed, see `OneIndexConstraintCompute`
```
...and similar for the two-index case?

Reply:
agreed, changed  to simpler individual comments